analysis/analysis.py

```diff
@@ -0,0 +1,91 @@
+from matplotlib import pyplot as plt
+from data.data_request import Token_Pair
+from data.market import Automted_Market_Maker
+from simulation.simulation import Simulation
+import pandas as pd
+
+
+def get_initial_drawdown(series: pd.Series) -> float:
+    """Computes the maximum drawdown from the first data point in the series.
+
+    Args:
+        series (pd.Series): A time series object containing price return data.
+
+    Returns:
+        float: The highest drawdown from the first data point in the series
+    """
+    return (min(series) / series[0] - 1)
+
+
+class Analysis:
+    """This class implements methods to analyse the results of a given simulation
+    """
+
+    def __init__(self, simulation: Simulation):
+        self._simulation = simulation
+
+    @property
+    def simulation(self) -> Simulation:
+        return self._simulation
+
+    # TODO: Refactor this, maybe even into a separate class for plots.
+    def plot_returns(self, data_label, title, type="hist"):
```

Comment:
Minor tip that can be applied in other places: Can use an [enum](https://docs.python.org/3/library/enum.html#creating-an-enum) instead of a general string for the `type`. Users will then know with certainty which plots are supported.

Reply:
You mean something like this?
```
class Plot(Enum):
    HIST = "hist
    LINE = "line"
    
analysis.plot_returns("label", "title", type = Plot.HIST)
```